config/config.php

```diff
@@ -12,32 +12,10 @@
     $containerConfigurator->import(__DIR__ . '/services-packages.php');
     $containerConfigurator->import(__DIR__ . '/parameters.php');
 
-    // rector root
-    $containerConfigurator->import(__DIR__ . '/../vendor/rector/rector-symfony/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../vendor/rector/rector-nette/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../vendor/rector/rector-laravel/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../vendor/rector/rector-phpunit/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../vendor/rector/rector-cakephp/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../vendor/rector/rector-doctrine/config/config.php', null, 'not_found');
-
-    // rector sub-package
-    $containerConfigurator->import(__DIR__ . '/../../rector-symfony/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../../rector-nette/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../../rector-laravel/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../../rector-phpunit/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../../rector-cakephp/config/config.php', null, 'not_found');
-    $containerConfigurator->import(__DIR__ . '/../../rector-doctrine/config/config.php', null, 'not_found');
-
-    // only for dev
-    // only for dev
-    if (class_exists(RectorGeneratorBundle::class)) {
-        $containerConfigurator->import(
-            __DIR__ . '/../vendor/rector/rector-generator/config/config.php',
-            null,
-            'not_found'
-        );
-        // 3rd party dependency
-        $containerConfigurator->import(__DIR__ . '/../../rector-generator/config/config.php', null, 'not_found');
+    $extensionConfigResolver = new ExtensionConfigResolver();
+    $configFiles = $extensionConfigResolver->provide();
+    foreach ($configFiles as $configFile) {
+        $containerConfigurator->import($configFile->getRealPath(), null, 'not_found');
```

Comment:
Ignoring errors should not be necessary, but it is a safety net, if someone / something deletes moves the config file accidentally and not redefining it in the composer.json extra section.

Reply:
This might hide unwanted bugs. How would you report missing config to package owners?